infra/capi/cluster-capi-infrastructure-ironic/cluster-api-infrastructure-ironic.yaml

```diff
@@ -0,0 +1,204 @@
+apiVersion: v1
```

Comment:
@vjayaramrh Can we add this as an independent yaml? It will be easy to read.

Reply:
The file is generated by the command `kustomize build ironic-deployment/default > cluster-api-infrastructure-ironic.yaml` and it would be error prone to do separate out manually and maintain it.